xml/System.Collections.Concurrent/ConcurrentDictionary`2+AlternateLookup`1.xml

```diff
@@ -128,10 +127,8 @@
       <Docs>
         <param name="key">The alternate key of the value to get or set.</param>
         <summary>Gets or sets the value associated with the specified alternate key.</summary>
-        <value>
-          <para>The value associated with the specified alternate key. If the specified alternate key is not found, a get operation throws</para>
-          <para>a <see cref="T:System.Collections.Generic.KeyNotFoundException" />, and a set operation creates a new element with the specified key.</para>
-        </value>
+        <value>The value associated with the specified alternate key. If the specified alternate key is not found, a get operation throws</para>
+          <para>a <see cref="T:System.Collections.Generic.KeyNotFoundException" />, and a set operation creates a new element with the specified key.</value>
```

Comment:
This produced invalid XML and did not correctly eliminate `<para>` and `</para>` tags.

```suggestion
        <value>
            The value associated with the specified alternate key. If the specified alternate key is not found, a get operation throws a <see cref="T:System.Collections.Generic.KeyNotFoundException" />, and a set operation creates a new element with the specified key.
        </value>
```